From c78a5d29b06181c91296368dba842291ab4767e3 Mon Sep 17 00:00:00 2001
From: Geoff Thorpe <Geoff.Thorpe@freescale.com>
Date: Wed, 14 Mar 2012 04:49:51 +0000
Subject: [PATCH 05/14] qbman/dpaa_eth/usdpaa: dynamic pool-channel/CGR
 allocation

From QorIQ-SDK-V1.2-SOURCE-20120614-yocto.iso

This should be squashed into commit:
    "qbman/dpaa_eth/usdpaa: dynamic pool-channel/CGR allocation"

Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 arch/powerpc/boot/dts/p4080ds-usdpaa.dts |    9 ---------
 arch/powerpc/boot/dts/p4080ds.dts        |   10 ----------
 2 files changed, 19 deletions(-)

diff --git a/arch/powerpc/boot/dts/p4080ds-usdpaa.dts b/arch/powerpc/boot/dts/p4080ds-usdpaa.dts
index 854dd9c..a5d2be6 100644
--- a/arch/powerpc/boot/dts/p4080ds-usdpaa.dts
+++ b/arch/powerpc/boot/dts/p4080ds-usdpaa.dts
@@ -66,7 +66,6 @@
 		ethernet@0 {
 			compatible = "fsl,p4080-dpa-ethernet-init", "fsl,dpa-ethernet-init";
 			fsl,bman-buffer-pools = <&bp7 &bp8 &bp9>;
-			fsl,qman-channel = <&qpool4>;
 			fsl,qman-frame-queues-rx = <0x50 1 0x51 1>;
 			fsl,qman-frame-queues-tx = <0x70 1 0x71 1>;
 		};
@@ -78,56 +77,48 @@
 		ethernet@2 {
 			compatible = "fsl,p4080-dpa-ethernet-init", "fsl,dpa-ethernet-init";
 			fsl,bman-buffer-pools = <&bp7 &bp8 &bp9>;
-			fsl,qman-channel = <&qpool4>;
 			fsl,qman-frame-queues-rx = <0x54 1 0x55 1>;
 			fsl,qman-frame-queues-tx = <0x74 1 0x75 1>;
 		};
 		ethernet@3 {
 			compatible = "fsl,p4080-dpa-ethernet-init", "fsl,dpa-ethernet-init";
 			fsl,bman-buffer-pools = <&bp7 &bp8 &bp9>;
-			fsl,qman-channel = <&qpool4>;
 			fsl,qman-frame-queues-rx = <0x56 1 0x57 1>;
 			fsl,qman-frame-queues-tx = <0x76 1 0x77 1>;
 		};
 		ethernet@4 {
 			compatible = "fsl,p4080-dpa-ethernet-init", "fsl,dpa-ethernet-init";
 			fsl,bman-buffer-pools = <&bp7 &bp8 &bp9>;
-			fsl,qman-channel = <&qpool4>;
 			fsl,qman-frame-queues-rx = <0x5a 1 0x5b 1>;
 			fsl,qman-frame-queues-tx = <0x7a 1 0x7b 1>;
 		};
 		ethernet@5 {
 			compatible = "fsl,p4080-dpa-ethernet-init", "fsl,dpa-ethernet-init";
 			fsl,bman-buffer-pools = <&bp7 &bp8 &bp9>;
-			fsl,qman-channel = <&qpool4>;
 			fsl,qman-frame-queues-rx = <0x5c 1 0x5d 1>;
 			fsl,qman-frame-queues-tx = <0x7c 1 0x7d 1>;
 		};
 		ethernet@6 {
 			compatible = "fsl,p4080-dpa-ethernet-init", "fsl,dpa-ethernet-init";
 			fsl,bman-buffer-pools = <&bp7 &bp8 &bp9>;
-			fsl,qman-channel = <&qpool4>;
 			fsl,qman-frame-queues-rx = <0x5e 1 0x5f 1>;
 			fsl,qman-frame-queues-tx = <0x7e 1 0x7f 1>;
 		};
 		ethernet@7 {
 			compatible = "fsl,p4080-dpa-ethernet-init", "fsl,dpa-ethernet-init";
 			fsl,bman-buffer-pools = <&bp7 &bp8 &bp9>;
-			fsl,qman-channel = <&qpool4>;
 			fsl,qman-frame-queues-rx = <0x60 1 0x61 1>;
 			fsl,qman-frame-queues-tx = <0x80 1 0x81 1>;
 		};
 		ethernet@8 {
 			compatible = "fsl,p4080-dpa-ethernet-init", "fsl,dpa-ethernet-init";
 			fsl,bman-buffer-pools = <&bp7 &bp8 &bp9>;
-			fsl,qman-channel = <&qpool4>;
 			fsl,qman-frame-queues-rx = <0x62 1 0x63 1>;
 			fsl,qman-frame-queues-tx = <0x82 1 0x83 1>;
 		};
 		ethernet@9 {
 			compatible = "fsl,p4080-dpa-ethernet-init", "fsl,dpa-ethernet-init";
 			fsl,bman-buffer-pools = <&bp7 &bp8 &bp9>;
-			fsl,qman-channel = <&qpool4>;
 			fsl,qman-frame-queues-rx = <0x66 1 0x67 1>;
 			fsl,qman-frame-queues-tx = <0x86 1 0x87 1>;
 		};
diff --git a/arch/powerpc/boot/dts/p4080ds.dts b/arch/powerpc/boot/dts/p4080ds.dts
index 03319bf..5cc580f 100644
--- a/arch/powerpc/boot/dts/p4080ds.dts
+++ b/arch/powerpc/boot/dts/p4080ds.dts
@@ -443,52 +443,42 @@
 
 		ethernet@0 {
 			compatible = "fsl,p4080-dpa-ethernet", "fsl,dpa-ethernet";
-			fsl,qman-channel = <&qpool1>;
 			fsl,fman-mac = <&enet0>;
 		};
 		ethernet@1 {
 			compatible = "fsl,p4080-dpa-ethernet", "fsl,dpa-ethernet";
-			fsl,qman-channel = <&qpool1>;
 			fsl,fman-mac = <&enet1>;
 		};
 		ethernet@2 {
 			compatible = "fsl,p4080-dpa-ethernet", "fsl,dpa-ethernet";
-			fsl,qman-channel = <&qpool1>;
 			fsl,fman-mac = <&enet2>;
 		};
 		ethernet@3 {
 			compatible = "fsl,p4080-dpa-ethernet", "fsl,dpa-ethernet";
-			fsl,qman-channel = <&qpool1>;
 			fsl,fman-mac = <&enet3>;
 		};
 		ethernet@4 {
 			compatible = "fsl,p4080-dpa-ethernet", "fsl,dpa-ethernet";
-			fsl,qman-channel = <&qpool1>;
 			fsl,fman-mac = <&enet4>;
 		};
 		ethernet@5 {
 			compatible = "fsl,p4080-dpa-ethernet", "fsl,dpa-ethernet";
-			fsl,qman-channel = <&qpool1>;
 			fsl,fman-mac = <&enet5>;
 		};
 		ethernet@6 {
 			compatible = "fsl,p4080-dpa-ethernet", "fsl,dpa-ethernet";
-			fsl,qman-channel = <&qpool1>;
 			fsl,fman-mac = <&enet6>;
 		};
 		ethernet@7 {
 			compatible = "fsl,p4080-dpa-ethernet", "fsl,dpa-ethernet";
-			fsl,qman-channel = <&qpool1>;
 			fsl,fman-mac = <&enet7>;
 		};
 		ethernet@8 {
 			compatible = "fsl,p4080-dpa-ethernet", "fsl,dpa-ethernet";
-			fsl,qman-channel = <&qpool1>;
 			fsl,fman-mac = <&enet8>;
 		};
 		ethernet@9 {
 			compatible = "fsl,p4080-dpa-ethernet", "fsl,dpa-ethernet";
-			fsl,qman-channel = <&qpool1>;
 			fsl,fman-mac = <&enet9>;
 		};
 	};
-- 
1.7.9.7

